### DISCUSSION TITLE
第6节课作业(4班)

### DISCUSSION THREAD

**Discussion by FlamingoPg** [InternLM/Tutorial#454] [作业]
请大家把第六节课作业放在这里哦~

**Comment by zhjunqin** [comment]
zhjunqin: https://github.com/zhjunqin/InternLM-tutorial/tree/master/Sixth

**Comment by onlyzer0** [comment]
onlyzer0:https://blog.csdn.net/qq_42907812/article/details/135758139?spm=1001.2014.3001.5502

**Comment by phbst** [comment]
https://thankful-archeology-738.notion.site/b444f63d4ffb4fbb9cd39932d2e7e305?pvs=4
